src/DynamoPackages/PackageLoader.cs

```diff
@@ -553,6 +564,14 @@ public Package ScanPackageDirectory(string directory, bool checkCertificates)
                     return discoveredPackage; // success
                 }
 
+                // Conflict invloving a built-in package
+                if (discoveredPackage.BuiltInPackage || existingPackage.BuiltInPackage)
+                {
+                    // We show both packages but we mark the new one as unloaded.
+                    discoveredPackage.LoadState.SetAsUnloaded();
+                    Add(discoveredPackage);
```

Comment:
Is this `Add` call so that we can show it in the UI?

Reply:
yes. The `Add` call will add the package to the LocalPackages array....but with the state set to Unloaded (which will not allow loading into the VM)

Reply:
But won't the `discoveredPackage `already be added to the array some time earlier?

Reply:
discoveredPackage is the new package that we are trying to Load.
existingPackage is the package with the same name that was already loaded.

At line code 570, the discoveredPackage  has not yet been added to the LocalPackages list